Classes/Controller/Haiku/AjaxController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace T3docs\Examples\Controller\Haiku;
+
+use Psr\Http\Message\ServerRequestInterface;
+use T3docs\Examples\Utility\HaikuSeasonUtility;
+use TYPO3\CMS\Core\Http\PropagateResponseException;
+use TYPO3\CMS\Core\Localization\LanguageService;
+use TYPO3\CMS\Core\Localization\LanguageServiceFactory;
+
+class AjaxController
```

Comment:
```suggestion
final class AjaxController
```

You know, I like final classes ;-)